aci/resource_aci_l3extsubnet.go

```diff
@@ -169,6 +226,42 @@ func setL3ExtSubnetAttributes(l3extSubnet *models.L3ExtSubnet, d *schema.Resourc
 	return d, nil
 }
 
+func getAndSetl3extRsSubnetToProfileFromL3ExtSubnet(client *client.Client, dn string, d *schema.ResourceData) (*schema.ResourceData, error) {
+	l3extRsSubnetToProfileData, err := client.ReadRelationl3extRsSubnetToProfileFromL3ExtSubnet(dn)
+	if err != nil {
+		log.Printf("[DEBUG] Error while reading relation l3extRsSubnetToProfile %v", err)
+		d.Set("relation_l3ext_rs_subnet_to_profile", make([]map[string]string, 0, 1))
+		return nil, err
+	} else {
+		relParamList := make([]map[string]string, 0, 1)
+		relParams := l3extRsSubnetToProfileData.([]map[string]string)
+		for _, obj := range relParams {
+			relParamList = append(relParamList, map[string]string{
+				// obj["tnRtctrlProfileName"] is set to tDN in aci-go-client thus name is assigned to tn_rtctrl_profile_dn
+				"tn_rtctrl_profile_dn": obj["tnRtctrlProfileName"],
```

Comment:
Just for a note, you can create a Go Client PR to fix the naming issue for tnRtctrlProfileName to tDn.

Reply:
changed